packages/assets-controllers/package.json

```diff
@@ -78,11 +78,11 @@
     "typescript": "~4.8.4"
   },
   "peerDependencies": {
-    "@metamask/accounts-controller": "^10.0.0",
+    "@metamask/accounts-controller": "^11.0.0",
     "@metamask/approval-controller": "^5.1.2",
-    "@metamask/keyring-controller": "^12.2.0",
+    "@metamask/keyring-controller": "^13.0.0",
     "@metamask/network-controller": "^17.2.0",
-    "@metamask/preferences-controller": "^7.0.0"
+    "@metamask/preferences-controller": "^8.0.0"
   },
```

Comment:
Only peer dependencies with a major version change are updated due to https://github.com/MetaMask/core/pull/3881.